### PR TITLE
Better endpoint management

### PR DESCRIPTION
**Pull request by Su386yt**

### CODE REVIEW

**Comment by Su386yt** [comment]
closing #36

File: src/api/api.ts [hunk 1]
```diff
@@ -83,6 +100,31 @@ export function loadApi() {
...
+import useragentsJson from '../api/config/useragents.json'
+
+function loadEndpointData() {
+  DEFAULT_ENDPOINT_SETTINGS.ignoreAgentLogging = useragentsJson.default.ignoreLogging
+  DEFAULT_ENDPOINT_SETTINGS.validUserAgents = useragentsJson.default.validAgents
+
+  const endpoints: any = useragentsJson.endpoints 
+  for (let i = 0; i < Object.keys(useragentsJson.endpoints).length; i++) {
+    const endpoint =  Object.keys(useragentsJson.endpoints)[i]
+    console.log(endpoint)
```
Comment (j10a1n15) [inline src/api/api.ts:112]: remove these console logs

**Review by j10a1n15 — APPROVED** [review]
lgtm

File: src/api/api.ts (hunk 1 above)
Comment (Su386yt) [reply src/api/api.ts]: Anything else?

**Review by j10a1n15 — APPROVED** [review]

**Pull request merged**